qa/tasks/deepsea.py

```diff
@@ -176,10 +146,9 @@ def setup(self):
             raise ConfigError("Unsupported deepsea install method '%s'"
                                                 % self.config['install'])
 
-        self.setup_salt()
-
     def begin(self):
         super(DeepSea, self).begin()
+        log.info("DeepSea task begin...")
```

Comment:
See comment for the _setup_ method

Reply:
@kshtsk This is intentional. The log message in base Task class is already there. I'm adding a separate INFO-level log message to the subclass so I can easily see from the log which commands are coming from the base Task class, and which ones from the subclass.